react/features/feedback/components/FeedbackButton.web.js

```diff
@@ -0,0 +1,28 @@
+/* global config */
+import React, { Component } from 'react';
+
+/**
+ * A Web Component which renders feedback button.
+ */
+export class FeedbackButton extends Component {
```

Comment:
@saghul, we have all your comments on the Conversation tab of the PR even if they're outdated.